test/support/protos/test_service_v3.pb.ex

```diff
@@ -31,6 +33,8 @@ defmodule TestserviceV3.Enum do
 end
 
 defmodule TestserviceV3.TestRequest.GEntry do
+  @moduledoc false
+
```

Comment:
This is a generated file.  Did you add these manually?  I don't see any changes to the mix task used to generate this.

If this is flagging your linter, we should update the rules to ignore this generated module rather than editing it

Reply:
newer protoc and other deps that aren't easily pinned introduced this behavior

Reply:
It automatically adds `@moduledoc false` by this command `protoc --elixir_out=plugins=grpc,gen_descriptors=true:./test/support/protos --proto_path=priv/protos/ test_service_v3.proto`. I will add flag `--elixir_opt=include_docs=true` to dismiss this next time.